P202.Training.Data/Repositories/UserRepository.cs

```diff
@@ -27,14 +28,42 @@ public void CreateUser(User user)
             }
         }
 
+        public void DeleteUser(User user)
+        {
+            using (var session = _sessionManager.OpenSession())
+            using (var transaction = session.BeginTransaction())
+            {
+                session.Delete(user);
```

Comment:
Out of curiosity: will this work? It seems that the user is not attached to the NHibernate session so I'm not sure